### ISSUE TITLE
Epic - Pass AA accessibility 

### ISSUE THREAD

**Issue by AmnestyAM** [aspirepress/documentation#45]
testing

**Comment by toderash** [comment]
Note that for text, I want to meet WCAG 2.x AAA standard, which is 7:1 contrast ratio. I think in all other contexts, AA will be fine. I expect we'll be putting that requirement into our brand guidelines, and note that there are instances that meet the spec for text at >=24px but fail at =<19px.
